easybuild/tools/options.py

```diff
@@ -562,6 +563,9 @@ def github_options(self):
             'github-user': ("GitHub username", str, 'store', None),
             'github-org': ("GitHub organization", str, 'store', None),
             'install-github-token': ("Install GitHub token (requires --github-user)", None, 'store_true', False),
+            'list-prs': ("List pull requests", str, 'store_or_None', 
+                         ":".join([GITHUB_LIST_PR_STATES[0], GITHUB_LIST_PR_ORDERS[0], GITHUB_LIST_PR_DIRECTIONS[1]]),
```

Comment:
@migueldiascosta I would avoid using indices in a list of constants... Instead, use a dedicated constant for the default value (and use that constant when defining the list of acceptable values)?

I would say `,` makes more sense as a separator (`:` is used for `PATH`-like variables).